src/analysis/complex/upper_half_plane/basic.lean

```diff
@@ -9,7 +9,6 @@ import analysis.complex.basic
 import group_theory.group_action.defs
 import linear_algebra.general_linear_group
 
-
 /-!
```

Comment:
Can you get rid of the stray line here?  (I don't know if you know about the "files changed" tab on github which makes this kind of small difference very visible)

Reply:
Fixed. Thanks for the tip! I had seen this, but I'd forgotten to check it